cclib/method/electrons.py

```diff
@@ -31,6 +31,20 @@ def __repr__(self):
         """Returns a representation of the object."""
         return "Electrons"
 
+    def alpha(self):
+        """Number of alpha electrons"""
+        homos = self.data.homos
+        if len(homos) == 1:
```

Comment:
I don't think you need this condition, since it's returning the same thing as below.

Reply:
Doh ^^' Painfully true.